.github/workflows/scripts/team-approval-check.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-env node */
+
+module.exports = {
+  check: async ({github, context, core, team_slug}) => {
```

Comment:
Can we add a JSDoc to here to not lose the context and have proper intellisense?